<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStuDialog</class>
 <widget class="QDialog" name="AddStuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>25</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="studentIDLabel">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studentIDLineEdit">
       <property name="maxLength">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="genderComboBox">
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ageLabel">
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ageSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>手机：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phoneLineEdit">
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="enrollmentTimeLabel">
       <property name="text">
        <string>入学时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="academyLabel">
       <property name="text">
        <string>学院：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="academyComboBox">
       <item>
        <property name="text">
         <string>计算机学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>外语学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>建筑学院</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>生命学院</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="specialityLabel">
       <property name="text">
        <string>专业：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="specialityComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="classLabel">
       <property name="text">
        <string>班级：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="GPALabel">
       <property name="text">
        <string>GPA：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="GPASpinBox">
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>62</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>studentIDLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>genderComboBox</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>phoneLineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>academyComboBox</tabstop>
  <tabstop>specialityComboBox</tabstop>
  <tabstop>classComboBox</tabstop>
  <tabstop>GPASpinBox</tabstop>
  <tabstop>saveBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
